### DISCUSSION TITLE
Deep nesting - Why and how to avoid

### DISCUSSION THREAD

**Discussion by giscus** [linkdotnet/Blog.Discussions#44] [General]
# Deep nesting - Why and how to avoid

... if you need more than 3 levels of indentation, you're screwed anyway, and should fix your program.
This is written in the Linux style guide. Let's see why they have that rule and how we can overcome deeply nested code.

https://steven-giesel.com/blogPost/61c2f923-c100-449b-b67c-563970a7b214?utm_source=csharpdigest&utm_medium=&utm_campaign=1499

**Comment by dougboone5** [comment]
So a premium customer who orders more than 1,000 units doesn't get the 0.9d discount?? Expect some nasty phone calls....

**Reply by linkdotnet** [reply]
Psst…. They don’t know.

But yeah maybe a could have picked a better example. The message stands the same .
